components/table/table-col-sort-button.js

```diff
@@ -122,8 +131,24 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 		const iconView = !this.nosort ?
 			html`<d2l-icon icon="${this.desc ? 'tier1:arrow-toggle-down' : 'tier1:arrow-toggle-up'}"></d2l-icon>` :
 			null;
+		const buttonTitle = this._getSortButtonTitle();
+		const buttonDescription = !this.nosort ? 'click to change sort order' : 'click to add sort order';
+
+		return html`<button aria-description="${buttonDescription}" aria-label="${buttonTitle}" title="${buttonTitle}" type="button"><slot></slot>${iconView}</button>`;
+	}
+
+	_getSortButtonTitle() {
+		const columnDataTypeWords = {
+			'words': ['A to Z', 'Z to A'],
+			'numbers': ['low to high', 'high to low'],
+			'dates': ['old to new', 'new to old']
+		};
```

Comment:
Same deal here, but you can store this data as a static private like:
```javascript
export class TableColSortButton extends FocusMixin(LitElement) {

static #columnDataTypeKeys = {
	words: ['components.table.aToZ', 'components.table.zToA'],
	numbers: ['components.table.lowToHigh', 'components.table.highToLow'],
	dates: ['components.table.oldToNew', 'components.table.newToOld']
}
```

All these langterms that get put into `en.js` should be the complete sentence/phrase, like, `Sorted high to low`.

Then back here, call `localize`:
```javascript
if (!this.nosort) {
	sortKey = TableColSortButton.#columnDataTypeKeys[this.columnDataType][Number(!this.desc)]
	return this.localize(sortKey);
	
}
return '';
```